src/conf.ml

```diff
@@ -71,7 +71,10 @@ let distros = Distro.(active_distros `X86_64 |> List.filter (fun d ->
   | `Linux | `Windows -> true
   | _ -> false))
 
-let arches_for ~distro = Distro.distro_arches Ocaml_version.Releases.latest distro
+let arches_for ~distro =
+  match distro with
+  | `Windows (`Msvc, _) -> [`X86_64]
```

Comment:
This was intended as a temporary fix - is there an issue tracking doing this properly?